src/libcore/iter/traits/iterator.rs

```diff
@@ -1366,8 +1350,7 @@ pub trait Iterator {
     /// [`Some(T)`] again. `fuse()` adapts an iterator, ensuring that after a
     /// [`None`] is given, it will always return [`None`] forever.
     ///
-    /// [`None`]: ../../std/option/enum.Option.html#variant.None
-    /// [`Some(T)`]: ../../std/option/enum.Option.html#variant.Some
+    /// [`Some(T)`]: Some
```

Comment:
I've seen this quite a few times now, maybe it would be a good feature for intra-doc links to add, same as https://github.com/rust-lang/rust/issues/62834?